changelog/db.changelog-1.0.xml

```diff
@@ -14,4 +14,4 @@
             <column name="name" type="VARCHAR(255)"/>
         </createTable>
     </changeSet>
-</databaseChangeLog>
+</databaseChangeLog>
```

Comment:
There's still a difference in this file that I don't think it will like as you can't change it once run.  Looks like it's that you've removed an empty line at the end.